Homework2/Homework2/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework2
+{
+    class Program
+    {
+            public static double FunctionValue(string x)
+            {
+                double i = Double.Parse(x);
+                return Math.Sqrt(Math.Pow(2, Math.Pow(i, i)) + Math.Pow(i, Math.Pow(2, i)) + Math.Pow(i, Math.Pow(i, 2)));
+            }

```

Comment:
Функция по заданному **числу** _x_ должна вычислять числовое значение.
 У вас параметр метода - строка. Преобразование из строки в число нужно делать в `Main`
```suggestion
            public static double FunctionValue(double x)
            {
                return Math.Sqrt(Math.Pow(2, Math.Pow(x, x)) + Math.Pow(x, Math.Pow(2, x)) + Math.Pow(x, Math.Pow(x, 2)));
            }

```